app/components/google-map.js

```diff
@@ -493,12 +493,15 @@ export default Ember.Component.extend(GoogleObjectMixin, {
    */
   initGoogleMap: on('didInsertElement', function () {
     var canvas;
+    var map;
     this.destroyGoogleMap();
     if (helpers.hasGoogleLib()) {
       canvas = this.$('div.map-canvas')[0];
-      this.createGoogleObject(canvas, null);
-      this.scheduleAutoFitBounds();
+      map = this.createGoogleObject(canvas, null);
     }
+    $(document).on('shown.bs.tab', function () { 
+      google.maps.event.trigger(map, 'resize'); 
+    });
```

Comment:
This would be triggered even if the map is not in a tab. Also this means you make the component do some specific stuff for TBS which is not good.

Also jQuery plugins MUST be wrapped to be used with EmberJS, so here the bug is because you did not wrap the tab plugin of TBS into an ember component/object/whatever so that the DOM isn't modified by another way but EmberJS view specific code